source/Dialog.h

```diff
@@ -118,10 +108,12 @@ Dialog::Dialog(T *t, void (T::*fun)(int), const std::string &text, int initialVa
 
 
 template <class T>
-Dialog::Dialog(T *t, void (T::*fun)(const std::string &), const std::string &text)
+Dialog::Dialog(T *t, void (T::*fun)(const std::string &), const std::string &text,
+		const std::string& initialValue)
 	: stringFun(std::bind(fun, t, std::placeholders::_1))
 {
 	Init(text);
+	input = initialValue;
```

Comment:
as is done for `stringFun`, initialize input directly with the initial value, rather than assign.

Reply:
Done.